### PR TITLE
Validations and model specs

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following ⚒️ 

- [ ] Set up a testing environment
- [ ] Used validations for models
- [ ] Wrote unit tests for models
- [ ] Used Rspec as a testing framework

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  validates :Name, presence: true
+  validates :PostsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```
Comment (mateo951) [inline app/models/user.rb]: User model has validations ✔

File: app/models/post.rb [hunk 2]
```diff
@@ -4,6 +4,11 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   after_save :update_post_counter
 
+  validates :Title, presence: true
+  validates :Title, length: { maximum: 250 }
+  validates :CommentsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :CommentsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```
Comment (mateo951) [inline app/models/post.rb]: `Post` model has validations 🔥

File: spec/models/like_spec.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+require_relative '../rails_helper'
+RSpec.describe Like, type: :model do
+  # tests go here
+end
```
Comment (mateo951) [inline spec/models/like_spec.rb]: Please be sure to write some tests for `Like` model as it is part of the requirements 🤓

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @PNdunguMaina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @PNdunguMaina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**